lib/cryptoexchange/exchanges/lykke/services/market.rb

```diff
@@ -0,0 +1,58 @@
+module Cryptoexchange::Exchanges
+  module Lykke
+    module Services
+      class Market < Cryptoexchange::Services::Market
+        class << self
+          def supports_individual_ticker_query?
+            false
+          end
+        end
+
+        def fetch
+          output = super(ticker_url)
+          pairs_output = HTTP.get(dictionary_url)
```

Comment:
Instead of making 2 queries, an additional the the dictionary_url (since its already done in the Pairs side)
Would it be possible to just directly/simply parse the ticker_url only.

Then when a pair is passed to find the Ticker, if its something it does not understand because it came from dictionary with a weird different base/target, it will just fail and return nothing